app/backend/graphql/interfaces/modifyInterface.ts

```diff
@@ -14,7 +14,7 @@ export const ModifyInterface = interfaceType({
         }
 
         if (!context.teamSlug) {
-          throw new GraphQLError('team slug not found')
+          throw new GraphQLError('team slug not found', {})
```

Comment:
What does this `{}` do? It fixes the depreciated GraphQLError?

Reply:
exactly